api/tests/integration/infrastructure/repositories/certification-center-membership-repository_test.js

```diff
@@ -509,4 +509,89 @@ describe('Integration | Repository | Certification Center Membership', function
       });
     });
   });
+
+  describe('#disableMembershipsByUserId', function () {
+    const now = new Date('2022-12-05');
+    let clock;
+
+    beforeEach(function () {
+      clock = sinon.useFakeTimers(now);
+    });
+
+    afterEach(function () {
+      clock.restore();
+    });
+
+    context('when there is multiple memberships for the specified user id', function () {
+      it('disabled all memberships of this user', async function () {
```

Comment:
```suggestion
      it('should disabled all memberships of this user', async function () {
```

Reply:
Je partirai plutôt vers `disables ...`. En faite le **should** ne devrait plus être utilisé 😎.
Le code qui a été écrit ne `devrait` pas faire cette action mais `doit` faire cette action

Reply:
![confident](https://user-images.githubusercontent.com/38167520/205890336-7b4d433b-f8f2-4953-b339-89d7ef134f26.gif)